src/lib/learn-client/api/api-types.ts

```diff
@@ -75,6 +75,12 @@ export interface ApiCollectionTutorial
 	tutorial: ApiTutorial
 }
 
+export interface ApiBookmark {
```

Comment:
```suggestion
export interface ApiBookmark extends BaseModel {
```
This data will have the `id` & `created_at` etc.

Reply:
Fixed in https://github.com/hashicorp/dev-portal/pull/828/commits/b2a8f3191373ca34985dd519b6ec12e609f5287f!